core-bundle/src/Resources/views/Backend/be_header_menu.html.twig

```diff
@@ -0,0 +1,6 @@
+{% trans_default_domain 'contao_default' %}
+<nav aria-label="{{ 'MSC.headerNavigation'|trans }}">
+    <a href="{{ app.request.requestUri }}#skipHeaderNav" class="invisible">{{ 'MSC.skipNavigation'|trans }}</a>
```

Comment:
don't you need to add the translation domain here?

Reply:
There is a `trans_default_domain 'contao_default'` at the top. 😉